docs/en/reference/configuration.rst

```diff
@@ -136,6 +140,8 @@ Here are details about what each configuration option does:
 +----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
 | check_database_platform    | no         | true                         | Whether to add a database platform check at the beginning of the generated code. |
 +----------------------------+------------+------------------------------+----------------------------------------------------------------------------------+
+| organize_migrations        | no         | ``none``                     | Whether to organize migration classes under year (``year``) or year and month (``year_and_month``) subdirectories. |
```

Comment:
The text is reflective of the original no-'none' option.  I recommend changing the description to

```
Whether to not organize migration classes (``none``), organize them by year (``year``), or organize them by year and month (``year_and_month``) using subdirectories.
```

Reply:
I agree, but that should be done in the 3.0.x branch. The aim of this PR it just to port into 3.1.x whatever is there.